### DISCUSSION TITLE
How to add Google Fonts to theme-ui?

### DISCUSSION THREAD

**Discussion by fermartz** [system-ui/theme-ui#1192] [General] [answered]
I have searched the docs but I can't find how to add Google fonts. 

Can anybody point me in the right direction?

Thanks in advance and props to jxnblk for creating this wonderful THEME-UI.

**Comment by lachlanjc** [comment, marked as answer]
Google Fonts integration is separate from Theme UI—depending on what framework you’re using (Next.js, Gatsby, CRA, Redwood, etc) it’ll be different. Look for your framework’s documentation on adding tags to `<head>`, then use Google Fonts’ provided `<link>` tag.

**Reply by fermartz** [reply]
@lachlanjc Thanks for the quick reply.

Ok I was confused. So, If I'm using Gatsby I would use for example this [plugin](https://medium.com/@matt.readout/using-google-fonts-in-your-gatsby-js-projects-b59a3abfba15), then just declare the font in theme-ui. Like this:

export default {
  fonts: {
    body: 'whatever-font,
    heading: 'whatever-font,
    monospace: 'whatever-font, monospace',
  },

Just in case anybody has the same question 😄

**Reply by hasparus** [reply]
@ferMartz Yup, this should work. Be sure to also check out [typefaces](https://github.com/KyleAMathews/typefaces) by Kyle Mathews (Gatsby's founder). You  can just stick `require('typeface-something')` in your `gatsby-browser.js`.

**Reply by fermartz** [reply]
@hasparus typefaces looks like a great alternative. Thanks!!!
